types_extractor/src/main/kotlin/extractor/MyPythonSourceRootConfigurator.kt

```
@@ -0,0 +1,211 @@
+            for (i in import.indices) {
+                if (import[i].importedQName == null) {
+                    continue
+                }
+
+                try {
+                    val resolved = import[i].multiResolve()
+                    if (resolved.isEmpty()) {
+                        totalUnresolved++
+                        println(import[i].importedQName)
+                    } else {
+                        totalResolved++
+                    }
+                } catch (e: NullPointerException) {
+                }
+            }
+        }
+
+        override fun visitPyFromImportStatement(node: PyFromImportStatement) {
+            super.visitPyFromImportStatement(node)
+            try {
+                val resolved = node.resolveImportSourceCandidates()
+                if (resolved.isNotEmpty()) {
+                    totalResolved++
+                } else {
+                    totalUnresolved++
+                    println(node.text)
+                }
+            } catch (e: NullPointerException) {
+            }
+        }
+    }
+
+    private class ImportVisitor(val root: VirtualFile) : PyRecursiveElementVisitor() {
+        lateinit var curFile: VirtualFile
+        val sourceRoots: MutableList<VirtualFile?> = mutableListOf()
+        var preResolved: Int = 0
+
+        override fun visitPyImportStatement(node: PyImportStatement) {
+            val import = node.importElements
+            val toResolve = MutableList(import.size) { true }
+
+            for (i in import.indices) {
+                if (import[i].importedQName == null) {
+                    continue
+                }
+
+                try {
+                    val resolved = import[i].multiResolve()
+                    if (resolved.isNotEmpty()) {
+                        preResolved++
+                        toResolve[i] = false
+                    }
+                } catch (e: NullPointerException) {
+
+                }
```

The empty catch seems suspicious. Where exactly can it happen?